src/unit_tests/wazuh_db/test_wdb.c

```diff
@@ -399,6 +403,79 @@ void test_wdb_exec_stmt_error(void **state) {
     cJSON_Delete(result);
 }
 
+/* Tests wdb_exec_stmt_silent */
+
+void test_wdb_exec_stmt_silent_success(void **state) {
```

Comment:
The function succeeds either if the result of the call to `wdb_step` is `SQLITE_DONE` as well as `SQLITE_ROW`. I think that we should have two test cases to validate both scenarios. Something like...

```
test_wdb_exec_stmt_silent_success_sqlite_done
test_wdb_exec_stmt_silent_success_sqlite_row
```